Source/USILifeSupport/LifeSupportUtilities.cs

```diff
@@ -4,6 +4,14 @@ namespace LifeSupport
 {
     public static class LifeSupportUtilities
     {
+        public static double SecondsPerMonth()
+        {
+            const double secsPerMinute = 60d;
+            const double secsPerHour = secsPerMinute * 60d;
+            double secsPerDay = GameSettings.KERBIN_TIME ? secsPerHour * 6d : secsPerHour * 24d;
+            return GameSettings.KERBIN_TIME ? secsPerHour * 38.6d : secsPerDay * 30.4d;
```

Comment:
Are Kerbin months 38.6 hours long?  I think the return value should be secsPerDay*30d in both cases (i.e. no "?" necessary).